src/processor/minidump.cc

```diff
@@ -2206,6 +2206,11 @@ string MinidumpModule::debug_identifier() const {
       memcpy(&guid, &cv_record_elf->build_id,
              std::min(cv_record_->size() - MDCVInfoELF_minsize,
                       sizeof(MDGUID)));
+      if (minidump_->swap()) {
```

Comment:
May be worth adding a comment on how this interacts with `guid_and_age_to_debug_id` on LE hosts (see the list in the initial comment).